src/lib/journey/config.store.ts

```diff
@@ -0,0 +1,72 @@
+import { readable, type Readable } from 'svelte/store';
+import { z } from 'zod';
+
+export const journeyItemSchema = z.object({
+  journey: z.string(),
+  key: z.string(),
+  match: z.string().array(),
+});
+export const journeysSchema = z.object({
+  default: journeyItemSchema,
+  forgotPassword: journeyItemSchema,
+  forgotUsername: journeyItemSchema,
+  login: journeyItemSchema,
+  register: journeyItemSchema,
+});
+
+export const partialJourneysSchema = journeysSchema.partial();
+
+export let configuredJourneys: Readable<z.infer<typeof journeyItemSchema>[]>;
+
+/**
+ * Change `any` here to something that actually works
+ */
+const defaultJourneys: any = {
+  forgotPassword: {
+    journey: 'ResetPassword',
+    key: 'forgotPassword',
+    match: ['#/service/ResetPassword'],
```

Comment:
These are the strings that we test for when a link is clicked. If an href matches one of these strings, then we know to internally handle the link and prevent default behavior. All other links are ignored and allowed through.

Reply:
Not sure I understand the meaning of the `match` property here - is this intended to be a URL path that users can also use to specify a given tree e.g. `https://sdkapp.example.com:8443/#/service/Registration`?

Reply:
Oh I think I get this now - the 'match' is the value we check against for the Page Node "Description"? Does this mean that these links won't appear unless there is a match in the description? Is there another way to enable them?

Reply:
Ok I think I understand this better now. Thanks for explaining it.